website/app/styles/doc-components/scroll-to-top.scss

```
@@ -0,0 +1,29 @@
+@use "../typography/mixins" as *;
+
+.doc-scroll-to-top {
+  position: fixed;
+  right: 40px;
+  bottom: 40px;
+  z-index: 99;
+  width: 44px;
+  height: 44px;
+  padding: 8px 8px 4px 8px;
+  background-color: rgba(52, 53, 54, 80%);
+  border: 2px solid transparent;
+  border-radius: 50%;
+  box-shadow: 0 8px 12px 0 rgba(37, 38, 45, 8%);
+  cursor: pointer;
+  transition: bottom 150ms linear;
```

position transitions are [inefficient](https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/), better to use transforms:
```suggestion
  transition: transform 150ms linear;
```

---

@heatherlarsen is it ok a linear transition?

---

Thanks for these suggestions @didoo! It's been a long time since I've done CSS animations, and I wasn't trying to dig too deeply into it. Do you have another suggestion we should try? 

cc @jorytindall @majedelass any preferences/ideas for how to improve this animation?

---

Is there something wrong with the current transition? It's not clear from the comments here.

---

I don't think there's anything "wrong" with it. I assume @didoo is proposing we do something more polished since "linear" is the default.

---

I did some _very_ scrappy testing with a few different easing methods and it's pretty hard to tell the difference. The length of the page does slightly effect the easing so in the case of a very long page (e.g., table), it's more apparent. But it does seem to smooth out a small amount using `ease-in-out` instead of `linear`.

---

Thanks @jorytindall! This is more polished. @didoo, if this is a conformant version, then I say we go with Jory's version.

---

yes, already changed to `ease-in-out`.